source/library/element-stub.js

```diff
@@ -0,0 +1,67 @@
+export default class ElementStub {
+	/**
+	 * Creates a new ElementStub instance. See {@link ElementStub#constructor}
+	 * @constructor
+	 * @private
+	 */
+	static create(...args) {
+		return new ElementStub(...args);
+	}
+
+	/**
+	 * Style proxy object
+	 * @type {Object}
+	 * @private
+	 */
+	style = {};
+
+	/**
+	 * Reference to currently active animationFrame
+	 * @type {Object}
+	 * @private
+	 */
+	frame = null;
+
+	/**
+	 * Creates a new ElementStub instance
+	 * @constructor
+	 * @param  {Node} element Node to stub
+	 * @param  {Window} [window=global.window] Global context to use
+	 * @return {ElementStub}         ElementStub instance
+	 * @private
+	 */
+	constructor(element, window = global.window) {
+		this.element = element;
+
+		const loop = () => {
+			this.render(this.element, this.style._style || this.style);
```

Comment:
Again, I don't get the style property. There is not initializing besides the empty object literal above. I would guess (just a guess) that its written to from outside of this class. That would be bad design and should be done through a setter. Otherwise you expose internal api (property names) as external api and you could not easily refactor it without breaking changes.